README.md

```diff
@@ -227,4 +227,14 @@ beforeEach(
     fixture.detectChanges();
   })
 );
+```
+
+## Using scripts/resources.js file
```

Comment:
Can you add this to the table of contents?

Reply:
Yes, I will add in table of contents.

Reply:
At the top of this file is a table of contents. Here's what that markdown looks like:

```
- [Running the Application](#running-the-application)
- [Ensuring Quality in the Application](#ensuring-quality-in-the-application)
- [Building Native Binaries](#building-native-binaries)
- [Development Workflow](#development-workflow)
```

There's an entry for each header, so if you add a new header you'll need to add it here as well.

Reply:
Added using scripts resources js file content in readme file